src/Platform/Microsoft.Testing.Platform.MSBuild/buildMultiTargeting/Microsoft.Testing.Platform.MSBuild.targets

```diff
@@ -195,6 +196,7 @@
           Condition=" '$(GenerateSelfRegisteredExtensions)' == 'True' " >
     <TestingPlatformAutoRegisteredExtensions AutoRegisteredExtensionsSourcePath="$(_AutoRegisteredExtensionsSourcePath)"
                                              AutoRegisteredExtensionsBuilderHook="@(TestingPlatformBuilderHook)"
+                                             RootNamespace="$(RootNamespace)"
```

Comment:
Is it valid for users to do `<RootNamespace></RootNamespace>`? In this case, our generated code will not compile.

Reply:
From looking up online, this is not supposed to be valid syntax but it can work when done in a target. I don't know if that's worth falling back to some default namespace or considering this as not supported. Any opinion?

Reply:
I'd say if it's possible for `RootNamespace` to be empty, then we should generate in global namespace (the old behavior). I see for example Roslyn is doing this:

https://github.com/dotnet/roslyn/blob/1b6cd2ab7a0862ea0c4d61b60d826429b2de2236/src/Workspaces/VisualBasicTest/Microsoft.CodeAnalysis.VisualBasic.Workspaces.UnitTests.vbproj#L9

Not entirely sure why this can be useful though.

Reply:
Handled now